### PR TITLE
[DPEDE-1225] Add large header option to portal

### PR DESCRIPTION
**Pull request by emerson-pereira**

### CODE REVIEW

**Comment by emerson-pereira** [comment]
@jllr @josecarlosgonzalezv

Need advise:

- for portal, header were already large size, I set all header to base size and set the new one only to large
- regarding the dropdown menus, there's no option to remove chevron

**Comment by emerson-pereira** [comment]
> @jllr @josecarlosgonzalezv
> 
> Need advise:
> 
> * for portal, header were already large size, I set all header to base size and set the new one only to large
> * regarding the dropdown menus, there's no option to remove chevron

- Confirmed all headers should be large for portal
- Dropdown menu styles applied

File: src/chi/components/header/header.scss [hunk 1]
```diff
@@ -557,6 +570,21 @@
...
+    .chi-header__navbar {
+      .chi-tabs {
+        &:not(.-vertical) {
+          &.-sm {
+            &:not(.-vertical):not(.-responsive) {
+              height: 3rem;
+            }
+          }
+        }
+      }
```
Comment (jllr) [inline src/chi/components/header/header.scss:587]:
```suggestion
  &.-xl {
    .chi-header__navbar {
      .chi-tabs {
        &:not(.-vertical) {
          &.-sm {
            &:not(.-vertical):not(.-responsive) {
              height: 3rem;
            }
          }
        }
      }
```

File: src/chi/components/header/header.scss [hunk 2]
```diff
@@ -482,6 +489,16 @@
             height: 2rem;
           }
         }
+
+        &:not(.-vertical) {
+          &.-sm {
+            .chi-dropdown {
+              .chi-dropdown__trigger {
+                padding: 1rem 0;
```
Comment (jllr) [inline src/chi/components/header/header.scss:497]:
```suggestion
                padding: 0.5rem 0;
```

**Review by jllr — CHANGES_REQUESTED** [review]
Also, a visual test for lg size with dropdown menu should be created (similar to this: XL Header with Navbar)

**Comment by emerson-pereira** [comment]
> Also, a visual test for lg size with dropdown menu should be created (similar to this: XL Header with Navbar)

@jllr Visual test created

**Comment by jllr** [comment]
@emerson-pereira I will chek the chevron update when the PR builds. Thanks!

File: src/chi/components/header/header.scss [hunk 3]
```diff
@@ -568,17 +595,26 @@
...
-          left: 1rem;
-          position: absolute;
-          top: 0;
-          width: 1rem;
+  &.-lg,
+  &.-xl {
+    .chi-header__navbar {
+      .chi-dropdown__menu {
+        .chi-dropdown__menu-item {
+          &::before {
```
Comment (jllr) [inline src/chi/components/header/header.scss:607]: &:before must be on dropdown__menu selector and not target to menu items. It is causing it renders several times

**Review by jllr — CHANGES_REQUESTED** [review]
Also change to xs size in tabs to reflect the same size we have in Lumen example. Then the tests will be fine

File: test/chi/components/header.pug [hunk 4]
```diff
@@ -298,6 +298,87 @@ h3 Header with mobile secondary menu
...
+              a.chi-dropdown__menu-item(href='#')
+                i.chi-icon.icon-user(aria-hidden='true')
+                span Item 1
+              a.chi-dropdown__menu-item(href='#')
+                i.chi-icon.icon-logout(aria-hidden='true')
+                span Item 2
+    nav.chi-header__navbar
+      .chi-header__start
+        .-d--none.-d-lg--flex
+          ul.chi-tabs.-inverse.-sm.-animated
```
[the remark this answers is not in the document]
Comment (emerson-pereira) [reply test/chi/components/header.pug]: @jllr done

**Review by josecarlosgonzalezv — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**